.nvmrc

```diff
@@ -0,0 +1 @@
+16
```

Comment:
Can you set this to `lts/gallium` that is the long term support version of node 16.